test/test_regression/test_regression.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import os.path
+import sys
+import argparse
+
+import numpy as np
+
+import glob
+import unittest
+
+sys.path.append(os.path.abspath(os.path.dirname(__file__)))
+from regressiontools import rtt
+
+def test_all_regression():
+  tests = glob.glob(os.path.join(os.path.dirname(__file__),"test_*"))
+  npe = glob.glob(os.path.join(os.path.dirname(__file__),"test*.py"))
+  for n in npe:
+    tests.remove(n)
```

Comment:
not really sure if you are removing everything or just something from the tests, please add a comment explaining why remove is necessary

Reply:
It removes the test file itself (has to be named test_*.py) from the list of directories with tests.